.gitignore

```diff
@@ -133,3 +133,4 @@ dist
 # Others
 .DS_Store
 .idea
+tmp
```

Comment:
what will be in the tmp directory?

Reply:
Nothing at the moment. I stored some private keys while integrating the OP client and I've added it to `gitignore`.